config/routes.rb

```diff
@@ -191,7 +191,11 @@
       end
     end
 
-    resources :order_imports, :only => [ :new, :create ]
+    resources :order_imports, only: [:new, :create] do
+      member do
+        get "error_report", to: "order_imports#error_report"
```

Comment:
This might look a little cleaner as a one-liner `get "error_report", to: "order_imports#error_report", on: :member`.